src/Form/Checkout/CheckoutPaymentType.php

```diff
@@ -42,6 +42,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ])
                 ->add('installments', HiddenType::class, [
                     'mapped' => false,
+                ])
+                ->add('issuer', HiddenType::class, [
```

Comment:
this is specific to mercadopago right? maybe add a comment here `specific to mercadopago`